static/app/views/insights/browser/webVitals/queries/useSampleWebVitalTrace.tsx

```diff
@@ -0,0 +1,116 @@
+import {decodeList} from 'sentry/utils/queryString';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {useLocation} from 'sentry/utils/useLocation';
+import type {ProjectData} from 'sentry/views/insights/browser/webVitals/components/webVitalMeters';
+import {SPANS_FILTER} from 'sentry/views/insights/browser/webVitals/queries/useSpanSamplesWebVitalsQuery';
+import {Referrer} from 'sentry/views/insights/browser/webVitals/referrers';
+import type {WebVitals} from 'sentry/views/insights/browser/webVitals/types';
+import decodeBrowserTypes from 'sentry/views/insights/browser/webVitals/utils/queryParameterDecoders/browserType';
+import {useSpans} from 'sentry/views/insights/common/queries/useDiscover';
+import {SpanFields, type SubregionCode} from 'sentry/views/insights/types';
+
+type Props = {
+  transaction: string;
+  webVital: WebVitals;
+  enabled?: boolean;
+  projectData?: ProjectData[];
+};
+
+function useSampleWebVitalTrace({
+  transaction,
+  projectData,
+  webVital,
+  enabled = true,
+}: Props) {
+  const location = useLocation();
+
+  let field: SpanFields | undefined;
+  switch (webVital) {
+    case 'cls':
+      field = SpanFields.CLS;
+      break;
+    case 'fcp':
+      field = SpanFields.FCP;
+      break;
+    case 'ttfb':
+      field = SpanFields.TTFB;
+      break;
+    case 'inp':
+      field = SpanFields.INP;
+      break;
+    case 'lcp':
+    default:
+      field = SpanFields.LCP;
+      break;
+  }
+
+  const browserTypes = decodeBrowserTypes(location.query[SpanFields.BROWSER_NAME]);
+  const subregions = decodeList(
+    location.query[SpanFields.USER_GEO_SUBREGION]
+  ) as SubregionCode[];
+  const p75Value = projectData?.[0]?.[`p75(measurements.${webVital})`];
+  const webVitalFilter = p75Value ? `measurements.${webVital}:>=${p75Value}` : '';
+
+  const search = new MutableSearch(SPANS_FILTER);
+  search.addFilterValue(SpanFields.TRANSACTION, transaction);
+  search.addStringFilter(webVitalFilter);
+  if (browserTypes) {
+    search.addDisjunctionFilterValues(SpanFields.BROWSER_NAME, browserTypes);
+  }
```

Comment:
imo its more inuitive and consistent if we conditionally add the filter instead of adding an empty string as a filter if p75Value doesnt exist
```suggestion
  if (p75Value) {
      search.addStringFilter(`measurements.${webVital}:>=${p75Value}`)
  }
  if (browserTypes) {
    search.addDisjunctionFilterValues(SpanFields.BROWSER_NAME, browserTypes);
  }
```